scripts/common.sh

```diff
@@ -241,6 +241,7 @@ _decode_base64_url () {
 decode_jwt () { _decode_base64_url $(echo -n $1 | cut -d "." -f ${2:-2}) | jq .; }
 
 build_go_plugin () {
+  gateway_image_tag=v4.0.0-rc11
```

Comment:
@davegarvey , I had to hard-code it for now because it's missing.

Reply:
I've changed this to use the newer method which uses a common function